app/Http/Controllers/Entity/RelationController.php

```diff
@@ -214,6 +213,9 @@ public function update(StoreRelation $request, Entity $entity, Relation $relatio
                 if (!empty($mode)) {
                     $redirect['mode'] = $mode;
                 }
+                if (request()->option) {
+                    $redirect['option'] = request()->option;
```

Comment:
Same here `request()->get('option')` make it clearer and appear less "magic", which is easier to read